packages/did-provider-ebsi/src/services/EbsiRPCService.ts

```diff
@@ -0,0 +1,157 @@
+ * @param {{ params: UpdateBaseDocumentParams[], id:number }} args
+ */
+export const updateBaseDocument = async (args: { params: UpdateBaseDocumentParams[]; id: number }): Promise<Response> => {
+  const { params, id } = args
+  const options = {
+    method: 'POST',
+    headers: new Headers({
+      Authorization: 'Bearer ' + token,
+    }),
+    body: JSON.stringify({
+      jsonrpc,
+      method: 'updateBaseDocument',
+      params,
+      id,
+    }),
+  }
+  return await (await fetch(baseUrl, options)).json()
+}
+
+/**
+ * Call to build an unsigned transaction to add a verification method. Requires an access token with "didr_write" scope.
+ * @param {{ params: AddVerificationMethodParams[], id:number }} args
+ */
+export const addVerificationMethod = async (args: { params: AddVerificationMethodParams[]; id: number }): Promise<Response> => {
+  const { params, id } = args
+  const options = {
+    method: 'POST',
+    headers: new Headers({
+      Authorization: 'Bearer ' + token,
+    }),
+    body: JSON.stringify({
+      jsonrpc,
+      method: 'addVerificationMethod',
+      params,
+      id,
+    }),
+  }
+  return await (await fetch(baseUrl, options)).json()
+}
+
+/**
+ * Call to build an unsigned transaction to add a verification relationship. Requires an access token with "didr_write" scope.
+ * @param {{ params: AddVerificationMethodRelationshipParams[], id: number }} args
+ */
+export const addVerificationMethodRelationship = async (args: {
+  params: AddVerificationMethodRelationshipParams[]
+  id: number
+}): Promise<Response> => {
+  const { params, id } = args
+  const options = {
+    method: 'POST',
+    headers: new Headers({
+      Authorization: 'Bearer ' + token,
+    }),
+    body: JSON.stringify({
+      jsonrpc,
+      method: 'addVerificationMethodRelationship',
+      params,
+      id,
+    }),
+  }
+  return await (await fetch(baseUrl, options)).json()
+}
+
+/**
+ * Call to send a signed transaction to the blockchain. Requires an access token with "didr_invite" or "didr_write" scope.
+ * @param {{ params: SendSignedTransactionParams[], id: number }} args
+ */
+export const sendSignedTransaction = async (args: { params: SendSignedTransactionParams[]; id: number }): Promise<Response> => {
+  const { params, id } = args
+  const options = {
+    method: 'POST',
+    headers: new Headers({
+      Authorization: 'Bearer ' + token,
+    }),
+    body: JSON.stringify({
+      jsonrpc,
+      method: 'sendSignedTransaction',
+      params,
+      id,
+    }),
+  }
+  return await (await fetch(baseUrl, options)).json()
+}
+
+/**
+ * Gets the DID document corresponding to the DID.
+ * @param {GetDidDocumentParams} args
+ * @returns a did document
+ */
+export const getDidDocument = async (args: GetDidDocumentParams): Promise<DIDDocument> => {
+  const { did, validAt } = args
+  if (!did) {
+    throw new Error('did parameter is required')
+  }
+  const query = `?${validAt && `valid_at=${validAt}`}`
```

Comment:
Do not use string interpolation within string interpolation

Reply:
Extracted nested string interpolation into ternary